### DISCUSSION TITLE
Delete in override Down not work when subclassing MigrationBase

### DISCUSSION THREAD

**Discussion by pedrostefanogv** [fluentmigrator/fluentmigrator#1368] [General] [answered]
Prezados estou iniciando um projeto com o fluent, porem, não consigo escrever a reversão da migração. A unção delete não esta habilitada
----------
Dear I am starting a project with fluent, however, I can't write the migration reversal. The delete anointing is not enabled

#####
 <TargetFramework>net5.0</TargetFramework>
####
<PackageReference Include="FluentMigrator" Version="3.2.9" />
<PackageReference Include="FluentMigrator.Runner" Version="3.2.9" />
 <PackageReference Include="FluentMigrator.Runner.Postgres" Version="3.2.9" />

![Captura de tela de 2020-11-05 18-04-00](https://user-images.githubusercontent.com/2353264/98296186-51eae680-1f91-11eb-9def-3078517dfbe1.png)

**Comment by jzabroski** [comment, marked as answer]
I see you are sub classing MigrationBase.

Please try Migration?

**Comment by jzabroski** [comment]
@p3s Closing - MigrationBase doesn't set the Delete Expression Root like Migration does. 
 https://github.com/fluentmigrator/fluentmigrator/blob/e82aafa20e6dbe3cefa221303fe23cf8bf59fffd/src/FluentMigrator/Migration.cs#L33
